docs/technical-reference.md

```diff
@@ -10,13 +10,13 @@ We will make sure that you have the latest version of Quilt,
 and walk you through the CloudFormation deployment.
 
 We recommend that all users do one or more of the following:
-* [Schedule a Quilt engineer](https://www.meetingbird.com/m/quilt-install)
+* [Schedule a Quilt engineer](https://calendly.com/d/d6w-fsk-gw6/quilt-engineering-team)
```

Comment:
For this link use a round robin with me, you, Kevin. Their will be many more times available that way.

Reply:
Done. Created a new round-robin for the three of us.